doc/modules/cassandra/pages/managing/operating/auto_repair.adoc

```diff
@@ -0,0 +1,410 @@
+= Auto Repair
```

Comment:
Regrettably the combination of me moving this file and adding ~200 lines causes it to be treated as a new file :(